Source/WebCore/platform/graphics/WOFFFileFormat.cpp

```diff
@@ -72,13 +72,13 @@ static bool readUInt16(SharedBuffer& buffer, size_t& offset, uint16_t& value)
 static bool writeUInt32(Vector<uint8_t>& vector, uint32_t value)
 {
     uint32_t bigEndianValue = htonl(value);
-    return vector.tryAppend(std::span { reinterpret_cast_ptr<uint8_t*>(&bigEndianValue), sizeof(bigEndianValue) });
+    return vector.tryAppend(unsafeMakeSpan(reinterpret_cast_ptr<uint8_t*>(&bigEndianValue), sizeof(bigEndianValue)));
```

Comment:
There's `WTF::asByteSpan(const T&)`:

```suggestion
    return vector.tryAppend(asByteSpan(bigEndianValue));
```